stream.go

```diff
@@ -106,22 +108,32 @@ func NewClientStream(ctx context.Context, desc *StreamDesc, cc *ClientConn, meth
 	if err != nil {
 		return nil, toRPCErr(err)
 	}
-	return &clientStream{
+	cs := &clientStream{
 		t:     t,
 		s:     s,
 		p:     &parser{s: s},
 		desc:  desc,
 		codec: cc.dopts.codec,
-	}, nil
+	}
+	if EnableTracing {
```

Comment:
under what situations can t.NewStream fail? Perhaps this should be higher up so you'll get a trace entry in certain failure situations?